### PR TITLE
add new nav dropdown options

### PR DESCRIPTION
**Pull request by QuincyLarson**

### CODE REVIEW

File: common/app/components/Nav/links.json [hunk 1]
```diff
@@ -1,23 +1,46 @@
...
-  "content": "Map",
-  "link": "/map",
-  "isReact": true
-},{
-  "content": "Shop",
-  "link": "/shop"
-}]
+[
+  {
+    "content": "Community",
```
Comment (Bouncey) [inline common/app/components/Nav/links.json]: Would it not make sense to have `map` take a 'higher priority' position over `community`?

Comment (QuincyLarson) [inline common/app/components/Nav/links.json]: @Bouncey we are getting rid of the map button completely once we move it over to the side. Here's what my prototype looks like: http://codepen.io/FreeCodeCamp/full/VKXyZJ

**Comment by Bouncey** [comment]
## LGTM :+1:

**Comment by systimotic** [comment]
@Bouncey @QuincyLarson 
Sorry for commenting on a merged PR 😅 

I noticed that in his commit, Berkeley also edit the view. ([link](https://github.com/freeCodeCamp/freeCodeCamp/commit/3a4e62da44a77b5b96e319259eb9f46f529d2430#diff-2c2495ea340f89a7a206279fe26be370))
Should that also be done for this change?

**Comment by QuincyLarson** [comment]
@systimotic Yes - I noticed this, too. Would you be interested in creating a PR for this?

**Comment by systimotic** [comment]
@QuincyLarson I am! #12723

**Pull request merged**